product_docs/docs/pgd/5/ddl.mdx

```diff
@@ -150,7 +150,7 @@ take a long time to acquire the lock. Hence it's preferable to run DDLs from a
 single node or the nodes that have nearly caught up with replication changes
 originating at other nodes.
 
-The second kind is known as a relation DML lock. This kind of lock is used when
+The second kind is known as a relation DML lock (or global DML lock). This kind of lock is used when
```

Comment:
```suggestion
The second kind is known as a global DML lock (or relation DML lock). This kind of lock is used when
```

Beyond this point, it is referred to as a global DML lock, and there's no explanation of the "relation DML" term, so global DML lock should come first for consistency.

Reply:
Cool :+1: